src/components/MenuList.jsx

```diff
@@ -19,16 +19,21 @@ class MenuList extends React.Component {
     const { products } = this.state
     let productList = products.map((product) => {
       return (
-        <>
-          <Divider hidden />
-          <Item>
-            <Item.Content key={product.id} cy-data={`product_id_${product.id}`}>
-              <Item.Header cy-data={`product_title_${product.id}`} >{product.title}</Item.Header>
-              <Item.Description cy-data={`product_description_${product.id}`} >{product.description}</Item.Description>
-              <Item.Extra cy-data={`product_price_${product.id}`} >{product.price}kr</Item.Extra>
-            </Item.Content>
-          </Item>
-        </>
+        <Item cy-data={`product_id_${product.id}`} key={product.id}>
+          <Item.Content >
+            <Item.Header cy-data={`product_title`} >{product.title}</Item.Header>
+            <Item.Description cy-data={`product_description`} >{product.description}</Item.Description>
+            <Item.Extra cy-data={`product_price`} >{product.price}kr</Item.Extra>
+            {this.props.authenticated &&
```

Comment:
you could extract the props like you do with the state, like `const { authenticated } = this.props`